src/memex/db/__init__.py

```diff
@@ -1,40 +1,26 @@
 # -*- coding: utf-8 -*-
 
-from sqlalchemy import MetaData
-from sqlalchemy.ext import declarative
+#: The base class used by memex's models.
+#:
+#: This must be configured by the including application using
+#: :py:func:`memex.db.set_base`.
+Base = None
```

Comment:
It might make sense to have a very minimal example app using memex soon. We're adding a lot of config here and there without experiencing how this would look like in a very minimal application that uses memex.